style/constants.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+from typing import Set
+
+# Constants for utils.world_languages
+
+WORLD_LANG_FILENAME = "world-lang.json"
+WORLD_LANG_PATH = Path(__file__).parents[1] / "/resources/"
```

Comment:
[nit] you're using this part (`Path(__file__).parents[1]`) a lot. Why don't you take that into a variable and use that variable in many places in this module?